libs/ledger-live-common/src/families/stacks/bridge/utils/misc.ts

```diff
@@ -1,6 +1,6 @@
 import { BigNumber } from "bignumber.js";
 import flatMap from "lodash/flatMap";
-import { Account, Address, Operation, OperationType } from "@ledgerhq/types-live";
```

Comment:
Why remove it and change the signature of the `getAddress` method? It is more synthetic

Reply:
this type { freshAddress, freshAddressPath } and getAddress function is now strictly internal to the coin, i didn't want to change it too much, but it's possible that coin could simplify even further this: there is no need to return Address anymore there, they could just afford to be a getAddress(path)=>address actually, since we don't have to push them into a freshAddresses array.
Anyway as it is not used externally, on types-live perspective there are no need to have this concept of 'Address' pair, so indeed took a direction to remove what's not useful from the type side.
we could always reintroduce the concept if we want in future, like if we want to reorganize the field together, but today in the Account type, they are just nested as part of Account.
also to be clear: there is no type regression on this, we still have the same type safety, it allows coins to do the type they want.